### ISSUE TITLE
ask for help

### ISSUE THREAD

**Issue by qiqi-7q** [LuminLiu/HierFL#10]
Hi liu，
    I'm very sorry to bother you at this time, but I encountered a problem when reproducing your code. When I conducted the experiment with the parameters you set, the results of the code were opposite to the conclusions in the paper. I'd like to ask why this situation occurred. For example, when k1 remains the same, the accuracy obtained from the code will increase as k2 increases. I sincerely hope to get your reply.

**Comment by qiqi-7q** [comment]
Sorry, when verifying the first conclusion, according to the set parameters, it will be found that when the product is the same, an increase in k2 will lead to an increase in accuracy or faster convergence. In this case, what is the x-axis? Is it the number of local updates?

**Comment by qiqi-7q** [comment]
Sorry, can this code be accelerated with a GPU? After I accelerated it with PyTorch, the training is still very slow. I think there might be something wrong with the environment configuration. So, could you please tell me approximately how long the training takes?
